tensorflow/python/data/kernel_tests/filter_dataset_op_test.py

```diff
@@ -176,6 +176,14 @@ def testReturnComponent(self):
       with self.assertRaises(errors.OutOfRangeError):
         sess.run(get_next)
 
+  def testParallelFilters(self):
+    dataset = dataset_ops.Dataset.range(10).filter(
+        lambda x: math_ops.equal(x % 2, 0))
+    iterators = [dataset.make_one_shot_iterator() for _ in range(10)]
+    next_elements = [iterator.get_next() for iterator in iterators]
+    with self.test_session() as sess:
+      self.assertEqual([0 for _ in range(10)], sess.run(next_elements))
```

Comment:
nit: can use `[0] * 10`

Reply:
Right. I’d rather not change this in a cherry pick though.

Reply:
SG.